src/pages/homepage/Global.Style.js

```diff
@@ -0,0 +1,93 @@
+import { createGlobalStyle } from "styled-components";
+
+const GlobalStyle = createGlobalStyle`
+  
+@font-face {
```

Comment:
Separate font from html style and import it in index.css

Reply:
eventually removed global styles